extension/page-detect.js

```diff
@@ -11,10 +11,14 @@ window.pageDetect = (() => {
 
 	const isRepoTree = () => isRepo() && /\/tree\//.test(getRepoPath());
 
+	const isIssueSearch = () => location.pathname.indexOf('/issues') === 0;
```

Comment:
Can you add unit tests for `isIssueSearch` and `isPrSearch` [here](https://github.com/sindresorhus/refined-github/blob/006eb4ba974dca63022b8487e55c088216f813ab/test/page-detect.js)? The mock for `location` we have in place should already handle `pathname`